packages/project-access/test/project/cap.test.ts

```diff
@@ -78,7 +78,184 @@ describe('Test getCapModelAndServices()', () => {
         clearGlobalCdsModulePromiseCache();
     });
 
-    test('Get valid model and services, mock cds with local cds from devDependencies', async () => {
+    test('Get valid model and services, mock cds with local cds from devDependencies Updated API available in @sap/cds: 7.8.0', async () => {
+        // Mock setup
+        const cdsMock = {
+            env: {
+                'for': () => ({
+                    folders: {
+                        app: 'APP',
+                        db: 'DB',
+                        srv: 'SRV'
+                    }
+                })
+            },
+            load: jest.fn().mockImplementation(() => Promise.resolve('MODEL')),
+            compile: {
+                to: {
+                    serviceinfo: jest.fn().mockImplementation(() => [
+                        {
+                            'name': 'Forwardslash',
+                            'endpoints': [
+                                {
+                                    'path': 'odata/service/with/forwardslash/',
+                                    'kind': 'odata'
+                                }
+                            ]
+                        },
+                        {
+                            'name': 'Backslash',
+                            'endpoints': [
+                                {
+                                    'path': '\\odata\\service\\with\\backslash/',
+                                    'kind': 'odata'
+                                }
+                            ]
+                        },
```

Comment:
Path is a web URI so I don't this backslash is valid.

Reply:
Thank you, it's a good point. I might leave this test there for now.